lib/tasks/push-to-space/push-to-space.js

```diff
@@ -163,7 +165,8 @@ export default function pushToSpace ({
         )
           .then(removeEmptyEntities)
           .then((assetsToProcess) => {
-            return assets.processAssets(assetsToProcess)
+            // TODO: What are the default values of these parameters? Do we need defaults?
```

Comment:
hm I'd prefer to not send them if they aren't present in `option`s object. So if the user doens't provide them, we don't send them. (and let the cma provide it's own defaults, like I mentioned above).

There's isn't really a pretty way to do this, though. Maybe you can come up with something better:

```
const processingOptions = Object.assign({}, timeout && {timeout}, retryLimit && {retryLimit})
return assets.processAssets(assetsToProcess, processingOptions)...
```

Reply:
Thanks, makes sense! I will take this for now :)